$NetBSD: patch-aa,v 1.1 2009/05/23 21:45:45 sno Exp $

Patch out auto_install - this will be handled by pkgsrc

--- Makefile.PL.orig	2009-05-23 21:33:21.000000000 +0000
+++ Makefile.PL	2009-05-23 21:33:43.000000000 +0000
@@ -14,5 +14,4 @@
 
 test_requires 'HTTP::Request::AsCGI';
 
-auto_install;
 WriteAll;
